ext/php7/ddshared.c

```diff
@@ -0,0 +1,27 @@
+#include "ddshared.h"
+
+#include "datadog/container_id.h"
+#include "datadog/string.h"
+#include "ddtrace.h"
+#include "logging.h"
+
+ZEND_EXTERN_MODULE_GLOBALS(ddtrace);
+
+datadog_string *dd_container_id = NULL;
+
+void ddshared_minit(void) {
+    char *cgroup_file = NULL;
+    if (DDTRACE_G(test_cgroup_file) && DDTRACE_G(test_cgroup_file)[0]) {
+        cgroup_file = DDTRACE_G(test_cgroup_file);
+        ddtrace_log_debugf("Using test cgroup file: %s", cgroup_file);
+    }
+    dd_container_id = datadog_container_id(cgroup_file);
+}
```

Comment:
```suggestion
void ddshared_minit(void) {
    if (DDTRACE_G(cgroup_file) && DDTRACE_G(cgroup_file)[0]) {
        dd_container_id = datadog_container_id(DDTRACE_G(cgroup_file));
    }
}
```

This will break your tests, sorry.